app/build.gradle

```diff
@@ -327,7 +327,7 @@ android {
           jsonFile = file(project.property("jsonFile"))
         } else {
           // If no file provided use the test file
-          jsonFile = file(directory + "/info.json")
+          jsonFile = project.file("src/customexample/info.json")
```

Comment:
I have to change this because I was getting this error when git hook ran:
```
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/abdul/AndroidStudioProjects/kiwix-android/app/build.gradle' line: 332

* What went wrong:
A problem occurred evaluating project ':app'.
> /Users/abdul/.gradle/daemon/5.4.1/app/src/customexample/info.json (No such file or directory)
```

I'm not a gradle expert so I don't know if it's the correct way.
Lastly, I can see that lint is not ignoring the generated files but it should.

Reply:
I don't think this should be changed. Are you sure you are using kiwixDebug?

Reply:
Yes

Reply:
I think `name==kiwix` should be breaking you out of the function. You can put breakpoints in there to debug.

Reply:
Tested it. It's not the case.

Reply:
Ok, then you probably have an extra folder somewhere you shouldn't. Look at the map variable.